modules/Ecommerce/DealDay/Handlers/UpdateDealDayHandler.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Modules\Ecommerce\DealDay\Handlers;
+
+use Modules\Ecommerce\DealDay\Commands\UpdateDealDayCommand;
+use Modules\Ecommerce\DealDay\Repositories\DealDayRepository;
+
+class UpdateDealDayHandler
+{
+    public function __construct(
+        private DealDayRepository $repository,
+    ) {
+    }
+
+    public function handle(UpdateDealDayCommand $updateDealDayCommand)
```

Comment:
The `handle` method should have a return type hint. Since it's performing an update operation and not returning anything, it should be `void`:

```suggestion
    public function handle(UpdateDealDayCommand $updateDealDayCommand): void
```